Model/GetItemsToDeductFromOrder.php

```diff
@@ -52,7 +52,7 @@ public function execute(Order $order): array
     {
         $itemsToOrder = [];
 
-        /** @var \Magento\Sales\Model\Order\Item $orderItem */
+        /** @var \Magento\Sales\Model\Order\Item|null $orderItem */
```

Comment:
Hmm, this doesn't make sense to me, this could imply that `$orderItem` can be `null`? In which case `$orderItem->getParentItem()` would fail if it was. Can you check whether this typehint is required to pass PHPStan and if so, can we figure out one that makes a bit more sense in the context of `getAllVisibleItems`?